Change Dataset used in the Display Overview Map sample (#365)
Esri/arcgis-maps-sdk-kotlin-samples: samples/display-overview-map/README.md

* MapView
* OverviewMap

## About the data

The data used in this sample is the [OpenStreetMap Tourist Attractions for North America](https://www.arcgis.com/home/item.html?id=addaa517dde346d1898c614fa91fd032) feature layer, which is scale-dependent and displays at scales larger than 1:160,000.

## Additional information

> This sample uses the [overview map toolkit component](https://github.com/Esri/arcgis-maps-sdk-kotlin-toolkit/tree/main/toolkit/geoview-compose#display-an-overviewmap). For information about setting up the toolkit visit the [developer guide doc](https://developers.arcgis.com/kotlin/toolkit/#reference-the-library-from-jfrog).

Reviewer: ```suggestion
This sample uses the [overview map toolkit component](https://github.com/Esri/arcgis-maps-sdk-kotlin-toolkit/tree/main/toolkit/geoview-compose#display-an-overviewmap). For information about setting up the toolkit visit the [developer guide doc](https://developers.arcgis.com/kotlin/toolkit/).
```

> This sample uses the overview map toolkit component, which requires the [toolkit](https://github.com/Esri/arcgis-runtime-toolkit-qt) to be cloned and set up locally. For information about setting up the toolkit, as well as code for the underlying component, visit the repository's [UI Tools page](https://github.com/Esri/arcgis-runtime-toolkit-qt/blob/main/uitools).

Reviewer 2: This refers to the QT toolkit instead of Kotlin toolkit. I suggest the following (note that the link to the overviewmap doc won't work yet since it's not on main yet):
```suggestion
This sample uses the [overview map toolkit component](https://github.com/Esri/arcgis-maps-sdk-kotlin-toolkit/tree/main/toolkit/geoview-compose#display-an-overviewmap). For information about setting up the toolkit visit the [developer guide doc](https://developers.arcgis.com/kotlin/toolkit/#reference-the-library-from-jfrog).
```
